contracts/programs/bubblegum/src/lib.rs

```diff
@@ -357,7 +361,10 @@ pub mod bubblegum {
 
         // @dev: seller_fee_basis points is encoded twice so that it can be passed to marketplace instructions, without passing the entire, un-hashed MetadataArgs struct
         let metadata_args_hash = keccak::hashv(&[message.try_to_vec()?.as_slice()]);
-        let data_hash = keccak::hashv(&[&metadata_args_hash.to_bytes(), &message.seller_fee_basis_points.to_le_bytes()]);
```

Comment:
Right above this block, check that `creators` is valid.

Best way to do this is create `assert_data_valid` method like in `token_metadata`
[(link)](https://github.com/metaplex-foundation/metaplex-program-library/blob/cac9cfd042cc5a16f50d635074555392d3b9044e/token-metadata/program/src/utils.rs#L35-L132)